Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -628,8 +631,12 @@ dmaeth100_alloc_rx::
    exx
    ex    af,af'
    lcall _pb_reserve
-   pop   af'
-   pop   de'
+   ex    af,af'

```

Comment:
Shouldn't need to make this change -- my reading of the pushes is that these pops are OK.  The compiler can use the `ALTD` prefix to `pop af'` but there isn't a simple way to `push af'`.